js/tootils/src/index.ts

```diff
@@ -27,9 +28,9 @@ export const test = base.extend<{ setup: void }>({
 });
 
 export async function wait_for_event(
-	component: SvelteComponentTyped,
+	component: SvelteComponent,
 	event: string
-) {
+): Promise<SpyFn> {
```

Comment:
I cleaned this up and got rid of a deprecated type and I also added a return type to the helper. I feel like i did this in another PR, will merge main and see.

Reply:
ok this was already done.